### PR TITLE
(DOCSP-10754): Add issues and workarounds page

### PR DESCRIPTION
**Pull request by cbush**
## Jira
https://jira.mongodb.org/browse/DOCSP-10754

## Staged
https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/issues-and-workarounds/reference/issues-and-workarounds.html

### CODE REVIEW

File: source/reference/issues-and-workarounds.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This page contains known issues and any possible workarounds.
```
Comment (nlarew) [inline source/reference/issues-and-workarounds.txt:18]: Maybe we could be a little more specific? Known issues with _what?_

Maybe:
```rst
This page specifies recommended workarounds for known issues that you may
encounter while developing with MongoDB Realm.
```

File: source/reference/issues-and-workarounds.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Please note that you can provide feedback on the documentation with the
+feedback form on the bottom right of each page. 
+
+If you have suggestions for the product itself or wish to track existing
+feature ideas, please check out the `MongoDB Feedback Engine for Realm
+<https://feedback.mongodb.com/forums/923521-realm/>`_.
```
Comment (nlarew) [inline source/reference/issues-and-workarounds.txt:25]: - Can we put this into an admonition?
- Is "MongoDB Feedback Engine for Realm" the blessed name? Feel kinda clunky and nerdy IMO.
  - Maybe: "check out the Realm feedback on the official MongoDB feedback site."

File: source/reference/issues-and-workarounds.txt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This page contains known issues and any possible workarounds.
+
+Please note that you can provide feedback on the documentation with the
+feedback form on the bottom right of each page.
```
Comment (nlarew) [inline source/reference/issues-and-workarounds.txt:21]: This feels like a non sequitur (this page lists issues/workarounds -> leave docs feedback here). Maybe we could give some lead-in context e.g.
- Rest assured, we know about these issues and are working to fix them so there's no need to spam us with more bug reports if you get them.
- That said, if the workarounds on this page don't work for you, please feel free to leave feedback using the docs widget.
- If you have suggestions for the product itself...

File: source/reference/issues-and-workarounds.txt [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+If you have suggestions for the product itself or wish to track existing
+feature ideas, please check out the `MongoDB Feedback Engine for Realm
+<https://feedback.mongodb.com/forums/923521-realm/>`_.
+
+Sync
+----
+
+Sync fails to enable shortly after cluster creation
+===================================================
```
Comment (nlarew) [inline source/reference/issues-and-workarounds.txt:31]: nit: we typically use `~` as our h3 underline
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: source/reference/issues-and-workarounds.txt [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+You may encounter strange behavior when enabling {+sync+} on a
+newly-created cluster. This happens because an SRV record hasn't
```
Comment (nlarew) [inline source/reference/issues-and-workarounds.txt:34]: Do we have any concrete examples of "strange behavior"?

File: source/reference/issues-and-workarounds.txt (hunk 3 above)
Comment (nlarew) [reply source/reference/issues-and-workarounds.txt:21]: Do we want to mention anything about "If you encounter a bug with MongoDB Realm that's not listed on this page, ..."?

File: source/reference/issues-and-workarounds.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/reference/issues-and-workarounds.txt:18]: +1

**Pull request merged**